challenge/public/index.html

```diff
@@ -13,5 +13,57 @@
 <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <style>
+        table, th, td {
+            border: 1px solid black;
+        }
+    </style>
+    <table id="signs">
```

Comment:
sort of the idea of react is to have js render most stuff instead of having to redo static html. try moving this into the `App.jsx` file. after that try to see if you can put it in its own file under `components` and then import it and use it in `App.jsx`.